### PR TITLE
Add CONTRIBUTING.md for "My First PR" instructions

### LINKED ISSUE #403: Add instructions for a first contribution
At project nights, we walk through how to fork, clone, and make a first contribution to the site, and I think it'd be helpful to document a list of steps in the repo

### PR DESCRIPTION
**Pull request by nickymarino**
Resolves #403 by adding a new markdown file with instructions on how to:

- Fork
- Clone
- Make changes
- Run tests/formatters
- Push
- Create a PR

I also add a callout in `README.md` right before the instructions on cloning the repo, since many of these steps can be applied right before that clone step.

The format of `CONTRIBUTING.md` comes from the [Atom CONTRIBUTING.md](https://github.com/atom/atom/blob/master/CONTRIBUTING.md). Edits and suggestions welcome!

### CODE REVIEW

**Review by elmq0022 — APPROVED** [review]
:beers:

**Review by sophm256 — CHANGES_REQUESTED** [review]
I appreciate this well written write up. 

There were a few minor typos that need changing and I had a suggestion.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Contributing to Chipy.org
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to the Chicago Python User Group website, which are hosted in the [Chicago Python User Group Organization](https://github.com/chicagopython). These are guidelines, not rules. Use your best judgement, and feel free to propose changes to this document in a pull request (PR).
```
Comment (sophm256) [inline CONTRIBUTING.md:5]: line 5, change `are` to `is`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    ```
+    $ git clone https://github.com/YOUR_USERNAME_HERE/chipy.org
+
+    $ cd chipy.org
+    ```
+3. Create a new branch with a short, descriptive name of the changes you'll make:
+    ```
+    $ git checkout -b BRANCH_NAME
+    ```
+4. Make changes to the branch to fix the issue you selected
```
Comment (sophm256) [inline CONTRIBUTING.md:34]: Line 4, add period to end of sentence.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    $ make test
+
+    $ make lint
+
+    $ make format
+    ```
+6. If any of the commands above fail, make changes to your code to successfully pass the three comamnds above.
+7. Once your changes pass the automated tests and formatters, add your changes to the branch and make a commit with a descriptive message:
+    ```
+    $ git add .
```
Comment (sophm256) [inline CONTRIBUTING.md:46]: Change line 7 to:

Once your changes pass the automated tests and formatters, add your changes to the branch. Using 'git status', double check that the files are the files that you want to commit. Then make a commit with a descriptive message.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+4. Make changes to the branch to fix the issue you selected
+5. Chipy.org uses Pytest, Pylint, and Black to encourage good software development techniques. Before you commit and push your changes, run our test suite, linter, and formatter:
+    ```
+    $ make test
+
+    $ make lint
+
+    $ make format
+    ```
+6. If any of the commands above fail, make changes to your code to successfully pass the three comamnds above.
```
Comment (sophm256) [inline CONTRIBUTING.md:43]: At line 43, change `comamnds` to `commands` . It's the second to last word.

File: CONTRIBUTING.md (hunk 1 above)
Comment (elmq0022) [reply CONTRIBUTING.md:5]: I'll go ahead and disagree with this one.  :wink: The word set is singular, therefore the word "is" is appropriate.  I agree with the the rest of your comments @sophm256.  Thanks for doing a more through review than me.

Comment (elmq0022) [reply CONTRIBUTING.md:5]: [reference](https://www.writewithjean.com/2012/11/18/is-set-singular-or-plural/#:~:text=In%20the%20US%2C%20collective%20nouns%20like%20set%20are,haven%E2%80%99t%20decided%E2%80%A6.%E2%80%9D%20The%20committee%20is%E2%80%A6%20The%20jury%20disagree.)

Comment (sophm256) [reply CONTRIBUTING.md:5]: @elmq0022 You and are in agreement that the word "is" should be with "set". That's why I asked @nickymarino to change `are` to `is` in line 5. Specifically the phrase `which are hosted` needs to be changed to `which is hosted`. :wink:

Comment (elmq0022) [reply CONTRIBUTING.md:5]: Looks like I read that wrong.  I didn't realize line 5 wrapped around that far.  :sweat_smile:

File: CONTRIBUTING.md (hunk 2 above)
Comment (nickymarino) [reply CONTRIBUTING.md:34]: @sophm256 line 4 is a blank line, did you mean somewhere else?

File: CONTRIBUTING.md (hunk 4 above)
Comment (nickymarino) [reply CONTRIBUTING.md:43]: Good catch!

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -40,16 +46,18 @@ Please follow these steps to make your first PR!
...
+    $ git status
+
     $ git commit -m "Adding a new feature to Chipy.org"
     ```
 8. Push your commit(s) to your repository fork:
     ```
     $ git push origin BRANCH_NAME
     ```
-9. Create a pull request by going to your repository fork on GitHub and clicking "Compare & pull request)
+9.  Create a pull request by going to your repository fork on GitHub and clicking "Compare & pull request)
```
Comment (sophm256) [inline CONTRIBUTING.md:62]: Line 62, please take out extra space before 'Create'. There should be a single space between '9.' and 'Create'. Also, please change the ')' to be a `"` at the end of this line.

File: CONTRIBUTING.md (hunk 2 above)
Comment (sophm256) [reply CONTRIBUTING.md:40]: Sorry, I meant please add period to the end of line 34 (which was step 4).

**Comment by sophm256** [comment]
Also, please add period at the end of last sentence in Step 7. 

Thank you Nicky for making all of these changes!

File: CONTRIBUTING.md (hunk 2 above)
Comment (nickymarino) [reply CONTRIBUTING.md:40]: Gotcha! Fix will be pushed in next commit

File: CONTRIBUTING.md (hunk 5 above)
Comment (nickymarino) [reply CONTRIBUTING.md:62]: Fixed 👍🏼

**Comment by nickymarino** [comment]
I added a colon instead of a period at the end of step 7 because of the commands following it.

And no problem! Thanks for the editing 😄

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -59,5 +59,5 @@ Please follow these steps to make your first PR!
     ```
     $ git push origin BRANCH_NAME
     ```
-9.  Create a pull request by going to your repository fork on GitHub and clicking "Compare & pull request)
+9. Create a pull request by going to your repository fork on GitHub and clicking "Compare & pull request")
```
Comment (sophm256) [inline CONTRIBUTING.md:62]: LIne 62, please change ')' to a period. I think this will be the last thing you have to change. Thanks! : )

Comment (nickymarino) [reply CONTRIBUTING.md]: Geez I should've caught that earlier tonight, guess I'm tired. Fixed now though :)

**Review by sophm256 — APPROVED** [review]

**Pull request merged**